client/src/lib.rs

```diff
@@ -151,10 +151,10 @@ pub struct TlsConfig {
 /// If using mTLS, both the client cert and private key must be specified, this contains them.
 #[derive(Clone)]
 pub struct ClientTlsConfig {
-    /// The certificate for this client
-    pub client_cert: Vec<u8>,
-    /// The private key for this client
-    pub client_private_key: Vec<u8>,
```

Comment:
I'd prefer not to rename the fields as all the language SDKs are going to have to deal with that change and while it's simple, I don't know that it's totally worth it.

That said, the comment changes are a great way to add clarity. Let's just do that.